### DISCUSSION TITLE
Multipoint pointers on the map

### DISCUSSION THREAD

**Discussion by HemboPagi** [directus/directus#17877] [Feature Requests]
Hello,

Not sure if this is a bug or I am just doing something wrong, but i can not get pointers visible on the map view when using MultiPoint type for the field. I can see that query is made and results retured (see screenshot).
<img width="1001" alt="Screenshot 2023-03-22 at 10 04 24" src="https://user-images.githubusercontent.com/95413906/226838093-27982cfc-6dcc-441b-adff-448b61f13edc.png">

When using Point as the data type all works nicely.

All the best
H

**Comment by connorwinston** [comment]
Hi! Just tested this and `MultiPoint` does indeed work from my testing using the below GeoJson in the field. I am closing this out as it is not a feature request. If you need additional assistance please visit the Discord or file a bug report!
```json
{
	"type": "MultiPoint",
	"coordinates": [
		[
			100,
			0
		],
		[
			101,
			1
		]
	]
}
```
